app/views/govuk_publishing_components/components/docs/document_list.yml

```diff
@@ -104,7 +104,7 @@ examples:
           text: 'School exclusion'
           path: '/government/publications/school-exclusion'
   with_context:
-    description: Context can be provided to render next to the item title. This will be used on /government/organisations to indicate if an organisation is hosted on a separate website or being moved to GOV.UK
+    description: Context can be provided to render next to the item title.
```

Comment:
Probably not for this PR, but was this added just for the org pages index? In which case we could probably remove the whole feature.

Reply:
We could but I'm not sure if it'll come in useful for any of the other org pages work we have left to do so I'm going to leave it for now. We can revisit it later if we think it will never be used